java-checks/src/test/files/checks/unused/UnusedMethodParameterCheck.java

```diff
@@ -203,30 +203,45 @@ void bar(ActionMapping mapping, ActionForm form, HttpServletRequest request, Htt
 
 class DocumentedMethod {
   /**
-   * @param firstArg
-   * @param secondArg
-   * @param fourthArg
+   * @param firstArg proper javadoc description
```

Comment:
This change of behavior of `UnusedMethodParameterCheck` needs to be documented in ticket title/summary as well.

Reply:
Updated ticket title and description to describe this behaviour change